postgres/datadog_checks/postgres/data/conf.yaml.example

```diff
@@ -161,6 +161,11 @@ instances:
     #
     # max_relations: 300
 
+    ## @param collect_relations_metrics - boolean - optional - default: false
+    ## If set to true, collects lock metrics.
+    #
+    # collect_relations_metrics: false
+
```

Comment:
This file is actually generated. You have to modify the spec like so here: https://github.com/DataDog/integrations-core/commit/ee97f13eeed95926cd5776949f1a4198ca3e7700#diff-49ce2a135c95e30effa64ca894cf499810631cee4b59c6f2a05a82ef42c0be2fR219-R231

Then run `ddev validate config postgres -s` to update the generated files.